snuba/utils/streams/profiler.py

```diff
@@ -4,12 +4,11 @@
 from pathlib import Path
```

Comment:
Up to you but I would consider providing the streaming library with a profiler. So I would move this one there.

Reply:
Will do